Docs/Configuration.tex

```diff
@@ -1469,42 +1469,41 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   This quirk requires \texttt{OC\_FIRMWARE\_RUNTIME} protocol implemented
   in \texttt{OpenRuntime.efi}.
 
-  \emph{Note}: This quirk can also be used as an ugly workaround to buggy UEFI
-  runtime services implementations that fail to write variables to NVRAM and
-  break the rest of the operating system.
+  \emph{Note}: This quirk can also be used as an ad hoc workaround for flawed UEFI
+  runtime services implementations that are unable to write variables to NVRAM
+  and results in operating system failures.
 
 \item
   \texttt{DiscardHibernateMap}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
   \textbf{Description}: Reuse original hibernate memory map.
 
-  This option forces XNU kernel to ignore newly supplied memory map and assume
-  that it did not change after waking from hibernation. This behaviour is required
-  to work by Windows, which mandates to
-  \href{https://docs.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-uefi#hibernation-state-s4-transition-requirements}{preserve}
+  This option forces the XNU kernel to ignore a newly supplied memory map and assume
+  that it did not change after waking from hibernation. This behaviour is required for
+   Windows to work. Windows mandates
```

Comment:
This change is incorrect. The option discards the new MemoryMap as it is not valid (apparently), but the reason it is allowed to do so is because all Windows-compatible computers must not change memory between boot and hibernate wake. The change in the wording makes it sound as the quirk is needed by windows when it is not.

Reply:
Restored "This behaviour is required to work by Windows" although no idea what that means!

Reply:
Windows requires the original memory map to be valid, so the sentence is here to explain why the quirk can be safe to apply.

Reply:
Thanks. That sentence is still not understandable grammatically but restored so we can progress.